### ISSUE TITLE
Scenario Outline cannot take multiple dynamic values

### ISSUE THREAD

**Issue by gemmadlou** [bencompton/jest-cucumber#75]
Thanks for your package. I'm finding it really awesome. I just had one issue. I'm not a Typescripter otherwise I would dare a PR.

Having the following outline:

```
Scenario Outline: Publishing a new release version <Version> to <NewVersion>
```

I was expecting the following type messages:

```
    ✓ Publishing a new release version 1.0.0 to 1.1.0
    ✓ Publishing a new release version 2.3.5 to 2.4.0
    ✓ Publishing a new release version 1.0.0 to 1.0.1
```

But instead, I got: 

```
Publishing a new release version
```

Here's my entire example:

```
Feature: Publishing

    Scenario Outline: Publishing a new release version <Version> to <NewVersion>

        Given a package version of <Version>
        And a commit message: <CommitMessage>
        When we publish
        Then the new version should be <NewVersion>
        And the new commit should be <NewCommitMessage>
        And the new git tag should be created with version <NewVersion>
        And s3 should contain the new artefact zip <Version>/app.zip

        Examples:
            | Version | NewVersion | CommitMessage     | NewCommitMessage       |
            | 1.0.0   | 1.1.0      | feat: New feature | Version 1.1.0 released |
            | 2.3.5   | 2.4.0      | feat: New feature | Version 2.4.0 released |
            | 1.0.0   | 1.0.1      | fix: New fix      | Version 1.0.1 released |
```

I saw that there is a difference between the .ts file and the dist js file that is produced:

By making the following changes directly to the js file, I finally got the outcome I wanted.
```js
var getOutlineDynamicTitle = function (exampleTableRow, title) {
    return title.replace(/<(\S*)>/g, (_, firstMatch) => {
        return exampleTableRow[firstMatch || ''];
    });
};
var parseScenarioOutlineExample = function (exampleTableRow, outlineScenario) {
    var outlineScenarioTitle = outlineScenario.title;
    var exampleKeyTitle = getOutlineDynamicTitle(exampleTableRow, outlineScenarioTitle);
    var title = outlineScenarioTitle;
    if (exampleKeyTitle) {
        title = exampleKeyTitle
    }
    return {
        title: title,
        steps: parseScenarioOutlineExampleSteps(exampleTableRow, outlineScenario.steps),
        tags: outlineScenario.tags,
    };
};
```

I apologise for butchering your code.

This is the original:

```ts
const getOutlineDynamicTitle = (exampleTableRow: any, title: string) => {
    return title.replace(/<(\S*)>/g, (_, firstMatch) => {
        return exampleTableRow[firstMatch || ''];
    });
};
```

The regex was different: `(\S*)`. And the `parseScenarioOutlineExample` function stripped out the scenario outline and replace it with the non-preg-repaced ones.

**Comment by bruno-morel** [comment]
was about to submit this issue. Same problem here for me.

**Comment by SirSkorpan** [comment]
The current master branch supports this, I tried with your example. I believe the preview version, 2.0.12, on npm matches the master branch.

**Comment by bencompton** [comment]
This should be fixed in #80, and is available in v2.0.13.
